src/components/screens/wallet/overview/accountInfo/actionBar.js

```diff
@@ -117,20 +117,22 @@ const ActionBar = ({
         </div>
       )
     }
-    <div className={styles.helperIcon}>
-      <Tooltip
-        className={`${styles.tooltipWrapper} ${styles.centerContent} ${isMultisignature ? styles.whiteBackground : ''}`}
-        position="bottom"
-        size="maxContent"
-        content={(
-          <DialogLink component={isMultisignature ? 'multisigAccountDetails' : 'multiSignature'}>
-            <Icon name="multiSignatureOutline" className={styles.multisigIcon} />
-          </DialogLink>
-        )}
-      >
-        <p>{isMultisignature ? t('View multisignature account details') : t('Register multisignature')}</p>
-      </Tooltip>
-    </div>
+    {address === host && (
```

Comment:
Should we display members of a multisig group that is not the one we've signed in with? If yes, then we should change this condition to always show `multisigAccountDetails` and only show `multiSignature` for the host.